### DISCUSSION TITLE
How can I set my server to public and not behind login?

### DISCUSSION THREAD

**Discussion by etdebruin** [mastodon/mastodon#20240] [General]
So sorry to post this here but I can't seem to find the right words to google a solution.

My Mastodon server (self hosted) used to be public I think, where people did not need to log in to see the timeline. I think I checked a box somewhere and now it's locked behind a login.

I want to set my timeline for my server to public.

Thank you in advance for pointing me to the setting/option.

**Comment by MrGeneration** [comment]
You can see the local and federated timeline without any authentication by calling `/public`.
Users can find that link via `See what's happening` on the about page of your instance:

![image](https://user-images.githubusercontent.com/6549061/200863104-e8b36503-590c-4168-b45b-9799c4aee4c4.png)

Note that this is only available if you have `Allow unauthenticated access to public timeline` enabled within the administration admin settings.

![image](https://user-images.githubusercontent.com/6549061/200863477-a47f9c55-aa2a-4569-87a2-e990d948c967.png)

**Reply by etdebruin** [reply]
Thank you @MrGeneration - is there a way that an unauthenticated session can redirect to /public instead of /about?

**Reply by MrGeneration** [reply]
I believe you can't.
Just had a look into the nginx log file - there's no way to tell if the user is logged in or not.

Possibly by checking for a specific cookie but that's hacky and may cause faults. I wouldn't do it personally.
